docs/crds/hostrule.md

```diff
@@ -197,7 +196,7 @@ The TCP Settings section is responsible for configuring Parent virtualservice sp
 The `tcpSettings` block, in addition to any other parameters provided in the HostRule, is only applied to Parent VSes and dedicated VSes. The `tcpSettings` block does not have any effect on child VSes.
 
 In order to consume TCP setting configurations for parent VSes, the HostRule must be matched to a Shared/Dedicated VS FQDN, using the existing `fqdn` field in HostRule. 
-Where dedicated VSes are created corresponding to a single application, Shared VSes would host multiple application FQDNs. Therefore, in order to apply a HostRule to a dedicated VS, users can simply provide the application FQDN in the HostRule `fqdn` field. For Shared VSes however, users can either provide the AKO programmed Shared VS FQDN (TODO: Provide link), or utilize the `fqdnType: Contains` parameter with the Shared VS name itself.
+Where dedicated VSes are created corresponding to a single application, Shared VSes would host multiple application FQDNs. Therefore, in order to apply a HostRule to a dedicated VS, users can simply provide the application FQDN in the HostRule `fqdn` field. For Shared VSes however, users can either provide the AKO programmed Shared VS FQDN, or utilize the `fqdnType: Contains` parameter with the Shared VS name itself.
```

Comment:
@akshayhavile The (TODO: provide link) is supposed to redirect to a documentation that covers this feature

mentioned in the CHANGELOG for AKO 1.6.1 https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/master/CHANGELOG.md
`Add autoFQDN to shared virtual services.`

I think we should add some documentation for this somewhere and provide the link here.